nomad/structs/job.go

```diff
@@ -97,6 +97,7 @@ type JobStatusesJob struct {
 	// ParentID is set on child (batch) jobs, specifying the parent job ID
 	ParentID         string
 	LatestDeployment *JobStatusesLatestDeployment
+	Stop bool // has the job been manually stopped?
```

Comment:
We'll want to add this to the equivalent `api` package struct as well.

Reply:
I actually never did add this HTTP API to the `api` package, partly because I hadn't fully committed to the structure of it, long-term, only its use in the UI.  arguably I should go ahead and do it, but at least for now, Phil doesn't need to account for it in this PR.

Reply:
Ok. Yeah out of scope for this PR then.